### PR TITLE
Update automationbot v2 address + bump addresses version

### PR DESCRIPTION
**Pull request by marcinciarka**
n/a

### CODE REVIEW

**Review by zerotucks — DISMISSED** [review]

File: packages/deploy-configurations/configs/optimism.conf.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ export const config: SystemConfig = {
       PositionCreated: {
         name: 'PositionCreated',
         deploy: false,
-        address: '0x8061c24823094E51e57A4a5cF8bEd3CCf09d316F',
+        address: '0x5743b5606e94fb534a31e1cefb3242c8a9422e5e',
```
Comment (paszkowskiDamian) [inline packages/deploy-configurations/configs/optimism.conf.ts]: Well on optimism I cannot see any contract here.

File: packages/deploy-configurations/configs/test/optimism.conf.ts [hunk 2]
```diff
@@ -96,7 +96,7 @@ export const config: SystemConfig = {
       PositionCreated: {
         name: 'PositionCreated',
         deploy: true,
-        address: '0x8061c24823094E51e57A4a5cF8bEd3CCf09d316F',
+        address: '0x5743b5606e94fb534a31e1cefb3242c8a9422e5e',
```
Comment (paszkowskiDamian) [inline packages/deploy-configurations/configs/test/optimism.conf.ts]: I guess same here

File: packages/deploy-configurations/configs/optimism.conf.ts (hunk 1 above)
Comment (zerotucks) [reply packages/deploy-configurations/configs/optimism.conf.ts]: https://optimistic.etherscan.io/address/0x8061c24823094e51e57a4a5cf8bed3ccf09d316f#code

It's legitimately the PositionCreated contract on Optimism. Address clash between AutomationBot on mainnet and PC action on optimism.

Comment (marcinciarka) [reply packages/deploy-configurations/configs/optimism.conf.ts]: Reverted changes to that file.

File: packages/deploy-configurations/configs/test/optimism.conf.ts (hunk 2 above)
Comment (marcinciarka) [reply packages/deploy-configurations/configs/test/optimism.conf.ts]: Reverted changes to that file.

**Review by paszkowskiDamian — APPROVED** [review]

**Pull request merged**